src/lib.rs

```diff
@@ -126,7 +127,14 @@ impl Getch {
         }
 
         match source.read(&mut buf) {
-            Ok(0) => Ok(Key::Null),
+            Ok(0) => {
+                #[cfg(windows)]
+                #[allow(overflowing_literals)]
```

Comment:
why is this needed? I don't see a reason for it...

Reply:
Are you talking about `#[allow(overflowing_literals)]`?
I got a warning when I tried to `&` the `i16` type with `0x8000`, but I allowed it because it gives the desired result.
However, when I thought about it carefully, I realized that if I set it to a negative number from the beginning, the warning does not appear. (`-0x8000`)

Reply:
Yeah, I mean the allow attribute, sorry for the missing info.
I'm not sure if I can follow your thoughts, but you can also use `#[cfg(...)]` Things on match arms. i think returning Z would break on Linux?

Reply:
The `read()` method returns `Ok(0)` when it receives `EOF`. Since `getch-rs` only supports standard input by key input, I think that `Ok(0)` pattern will only occur when <kbd>Ctrl</kbd> + <kbd>Z</kbd> is received.
Linux accepts <kbd>Ctrl</kbd> + <kbd>Z</kbd> as a normal keystroke instead of `EOF`, so it doesn't break.